### PR TITLE
PC-9863: add oft for gateexpress

### PR DESCRIPTION
**Pull request by DimaSorokin**

### CODE REVIEW

File: data/payment_providers.json [hunk 1]
```diff
@@ -18510,5 +18510,18 @@
       "kanata_sahakari_bank",
       "amazon_pay"
     ]
+  },
+  {
+    "status": "active",
+    "code": "gateexpress",
+    "vendor": "gateexpress",
+    "name": {
+      "en": "GateExpress",
```
Comment (TonyTramb) [inline data/payment_providers.json:18519]: image

File: data/vendors.json [hunk 2]
```diff
@@ -29257,5 +29257,17 @@
...
     }
+  },
+  {
+    "code": "gateexpress",
+    "status": "active",
+    "links": {
+      "website": "https://gate.express/"
+    },
+    "name": {
+      "en": "GateExpress",
```
Comment (TonyTramb) [inline data/vendors.json:29268]: Gate.Express

File: data/payment_providers.json (hunk 1 above)
Comment (DimaSorokin) [reply data/payment_providers.json]: +

File: data/vendors.json (hunk 2 above)
Comment (DimaSorokin) [reply data/vendors.json]: +

**Review by michaelkuzyk — APPROVED** [review]

**Pull request merged**